google_project/README.md

```diff
@@ -5,48 +5,49 @@ Sets up a single GCP project linked to a billing account plus management metadat
 
 | Name | Version |
 |------|---------|
-| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1.0 |
```

Comment:
I am not exactly sure why the formatting changed here, I am using `terraform-docs version v0.16.0 1f686b1 linux/amd64` and pre-commit.

Reply:
huh yeah, looks like it just made links

Reply:
which is cool, whatevs, live & let live as long as there are docs i guess